README.md

```diff
@@ -1,6 +1,13 @@
-New Relic's official MySQL instrumentation for use with the
-[Node agent](https://github.com/newrelic/node-newrelic). This module is a
-dependency of the agent and is installed with it by running:
+[![Community Project header](https://github.com/newrelic/open-source-office/raw/master/examples/categories/images/Community_Project.png)](https://github.com/newrelic/open-source-office/blob/master/examples/categories/index.md#community-project)
```

Comment:
I think we need an experimental header link here.
https://github.com/newrelic/node-newrelic-mysql/issues/25#issuecomment-668747382

Reply:
👍